gocd/templates/relay.jsonnet

```diff
@@ -15,6 +14,11 @@ local pipedream_config = {
       destination: 'relay',
     },
   },
+  rollback: {
+    material_name: 'relay_repo',
+    stage: 'deploy-production',
```

Comment:
@mattgauntseo-sentry with targeting this stage, it will also rollback `deploy-pops` which is the next stage, right?